jenkins/ParseArguments.cmake

```diff
@@ -0,0 +1,65 @@
+# Prevent double-definition if two projects use this script
+IF (NOT ParseArguments_INCLUDED)
```

Comment:
FYI this file is copied verbatim from tlm, so probably no need to review extensively.